modules/importer.js

```diff
@@ -277,6 +278,7 @@ class Importer {
             this.remoteControl.importRequestData();
             return {
                 response: await this.afterImport(result),
+                otjson_size: bytes(JSON.stringify(otJsonDoc)),
```

Comment:
Return the OT-JSON size as part of the response object (my suggestion is to write the dataset size in the result of the otJsonImporter importFile function, since that is what is passed to the afterImport function)